src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.cpp

```diff
@@ -317,24 +313,25 @@ bool FlightTaskAuto::_evaluateTriplets()
 	// 3. navigator originally only supports gps guided maneuvers. However, it now also supports some flow-specific features
 	// such as land and takeoff. The navigator should use for auto takeoff/land with flow the position in xy at the moment the
 	// takeoff/land was initiated. Until then we do this kind of logic here.
+	position_setpoint_triplet_s position_setpoint_triplet{};
+	_position_setpoint_triplet_sub.copy(&position_setpoint_triplet);
```

Comment:
Could this all be within `if (_position_setpoint_triplet_sub.update(&position_setpoint_triplet))`? Avoid unnecessarily zeroing and copying things around.